Cargo.toml

```diff
@@ -30,6 +30,7 @@ resolver = "2"
 inherits = "release"
 codegen-units = 1
 lto = true
+debug = true
```

Comment:
Are debug symbols missing otherwise?

Reply:
Ah, I did not actually want to commit this. This is something I need for better profiler output but we certainly don't want in the Docker image...